### DISCUSSION TITLE
proxmox

### DISCUSSION THREAD

**Discussion by Fredobedo** [Joshua-Riek/ubuntu-rockchip#67] [General]
Dear Orange PI 5 enthusiasts,

So far, I installed on top of the custom build of Joshua lots of other apps without any problems except for Proxmox which I couldn't find any way to install it correctly, most probable due to my incompetence :) 

The idea is that then, then we could give a try to install Windows 11 for ARM in a VM, etc.

NB: And Dear Joshua, thank you for giving your time to share your work, here, I really enjoy my Orange PI 5 thanks to your involvement, much appreciated.

Fred

**Comment by Joshua-Riek** [comment]
Isn't Proxmox an entire OS?

**Comment by Fredobedo** [comment]
You are most probably right, 
I just had a reading at this post https://www.reddit.com/r/Proxmox/comments/104z8pt/proxmox_arm_running_on_the_orange_pi_5_with_nvme/
He fist install the official Orange Pi debian image and then pimox7
-> thus I tried the install script for pimox7 on this build but with no luck.

**Comment by Joshua-Riek** [comment]
Interesting, ill have to try this later today. There could be an issue because it's made for Debian, not Ubuntu.
